docs/troubleshooting.md

```diff
@@ -73,6 +73,18 @@ Checking update Status
 Last Execution was 2023-03-20 12:20:03 UTC
 ```
 
+The same check that is done in the News admin settings can be done using occ too. 
+Adding the --check-elapsed option displays the time elapsed since the last execution,
+and if it's considered too long ago, a message will be displayed, and the command returns
+with exit code 2. This can be used in scripts to send an alert for example.
+```bash
+sudo -u www-data php ./occ news:updater:job --check-elapsed
```

Comment:
```suggestion
```console
$ sudo -u www-data php ./occ news:updater:job --check-elapsed
```

Reply:
What's changed here were copied verbatim from the other adjacent examples. I'm confused.

Reply:
`bash` suggests it's all a bash script.
`console` defines it as console in/output.

Might be the same as the other blocks, but `console` renders it nicer.